packages/@aws-cdk/aws-dynamodb-global/lib/aws-dynamodb-global.ts

```diff
@@ -0,0 +1,64 @@
+import dynamodb = require("@aws-cdk/aws-dynamodb");
+import cdk = require("@aws-cdk/cdk");
+import { LambdaGlobalDynamoDBMaker } from "./lambda-global-dynamodb";
+
+/**
+ * Properties for the mutliple DynamoDB tables to mash together into a
+ * global table
+ */
+export interface RegionalTablesProps extends cdk.StackProps, dynamodb.TableOptions {
+    /**
+     * Enforces a particular table name to share across Global DynamoDB tables
```

Comment:
Wording here feels off. "Enforces?" Ideally if we can copy & paste from official AWS docs that would be better. I would write something like "Name of the DynamoDB table to use across all regional tables. This is required for global tables".